components/organisms/RepositoriesTable/repositories-table.tsx

```diff
@@ -82,7 +87,7 @@ const RepositoriesTable = ({
 
       {isLoadedWithRepos &&
         listOfRepositories.map((item, index) => (
-          <RepoRow key={`${item.handle}/${item.name}/${index}`} topic={topic} repo={item} user={user} />
+          <RepoRow key={`${item.handle}/${item.name}/${index}`} topic={topic} repo={item} user={user} selected={selectedRepos.find((r) => r.id == item.id) != undefined} handleOnSelectRepo={handleOnSelectRepo} />
```

Comment:
I don't think the `selected` logic should be inline in the prop. Open up the `map` callback function, use a variable and pass it to the prop instead.